packages/typespec-azure-portal-core/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@azure-tools/typespec-azure-portal-core",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "tsc -p .",
+    "test": "mocha",
+    "test-official": "c8 mocha --forbid-only --reporter mocha-multi-reporters"
+  },
+  "keywords": [
+    "Azure",
+    "Portal",
+    "TypeSpec"
+  ],
+  "author": "",
+  "license": "MIT",
+  "type": "module",
+  "main": "dist/src/index.js",
+  "tspMain": "lib/main.tsp",
+  "peerDependencies": {
+    "@typespec/compiler": "~0.51.0"
+  },
+  "devDependencies": {
```

Comment:
We'll need to sort out which of these are bona fide dependencies and which are merely for testing. It seems like at a minimum typespec/compiler and azure-tools/typespec-azure-resource-manager need to be moved to the dependencies list.

Reply:
question on this!